src/network/taraxa_capability.cpp

```diff
@@ -1728,14 +1776,18 @@ void TaraxaCapability::broadcastPreviousRoundNextVotesBundle() {
   boost::shared_lock<boost::shared_mutex> lock(peers_mutex_);
   for (auto const &peer : peers_) {
     // Nodes may vote at different values at previous round, so need less or equal
-    if (peer.second->pbft_round_ <= pbft_current_round) {
+    if (!peer.second->syncing_ && peer.second->received_initial_status_ && peer.second->sent_initial_status_ &&
+        peer.second->pbft_round_ <= pbft_current_round) {
       std::vector<Vote> send_next_votes_bundle;
       for (auto const &v : next_votes_bundle) {
         if (!peer.second->isVoteKnown(v.getHash())) {
           send_next_votes_bundle.emplace_back(v);
         }
       }
       sendPbftNextVotes(peer.first, send_next_votes_bundle);
+      for (auto const &v : next_votes_bundle) {
+        peer.second->markVoteAsKnown(v.getHash());
```

Comment:
you should mark only what is in `send_next_votes_bundle`, but I would move all the marking to `sendPbftNextVotes`

Reply:
Fixed.